### ISSUE TITLE
Does the bn reestimation really work?

### ISSUE THREAD

**Issue by 666DZY666** [quic/aimet#1974]
There's something wrong with my verification, could you give me some reference data？

My experiment is as follows：
1, ptq —> 2, qat —> 3, bn reestimation —> 4, fold_all_batch_norms_to_scale

after step 3, there is a slight drop in accuracy；
after step 4, the accuracy is seriously reduced

**Comment by quic-mangal** [comment]
@quic-akhobare could you help answering this. Thanks

**Comment by quic-akhobare** [comment]
Hi @666DZY666 - interesting observations.

So after #4, you are still using AIMET QuantSim to do accuracy evaluation?

**Comment by 666DZY666** [comment]
yes, just follow: https://github.com/quic/aimet/blob/develop/Examples/torch/quantization/bn_reestimation.ipynb, after fold_all_batch_norms_to_scale, use sim.model to do accuracy evaluation. I think the pipeline is ok, but the result is bad.

**Comment by 666DZY666** [comment]
> Hi @666DZY666 - interesting observations.
> 
> So after #4, you are still using AIMET QuantSim to do accuracy evaluation?

**Comment by 666DZY666** [comment]
> Hi @666DZY666 - interesting observations.
> 
> So after #4, you are still using AIMET QuantSim to do accuracy evaluation?

yes, just follow: https://github.com/quic/aimet/blob/develop/Examples/torch/quantization/bn_reestimation.ipynb, after fold_all_batch_norms_to_scale, use sim.model to do accuracy evaluation. I think the pipeline is ok, but the result is bad.

**Comment by 666DZY666** [comment]
> Hi @666DZY666 - interesting observations.
> 
> So after #4, you are still using AIMET QuantSim to do accuracy evaluation?

I suspect this is caused by per-tensor quantization?

**Comment by quic-akhobare** [comment]
Ok I get it now. @quic-kyunggeu could you please advise on the right procedure here. So after the fold-to-scale, ideally we would just export the model and take it to target. Which is what the example shows.

However if someone wants to recreate the sim and evaluate, what would be the right way? I am guessing we need to reload the encodings that were saved?

**Comment by ShobithaShivakumar** [comment]
Closing this issue due to lack of activity. @666DZY666  please re-open this issue if you are still hitting issues with the latest aimet-torch (2.26). If you have further questions, feel free to join our[ Slack Community ](http://aihub.qualcomm.com/community/slack) specifically the #aimet channel to discuss easily!
